### PR TITLE
revert of #423

### PR DESCRIPTION
**Pull request by pavertomato**
this PR contains first step of revert - update of breast_density_auto_ml.ipynb

### CODE REVIEW

File: imaging/ml/ml_codelab/breast_density_auto_ml.ipynb [hunk 1]
```diff
@@ -114,22 +183,101 @@
...
    "outputs": [],
    "source": [
-    "%%bash -s {project_id} {location} {automl_bucket_name}\n",
-    "# Create bucket.\n",
-    "gsutil -q mb -c regional -l $2 $3\n",
+    "import time\n",
     "\n",
-    "# Allow Cloud Healthcare API to write to bucket.\n",
-    "PROJECT_NUMBER=`gcloud projects describe chc-tcia | grep projectNumber | sed 's/[^0-9]//g'`\n",
-    "SERVICE_ACCOUNT=\"service-${PROJECT_NUMBER}@gcp-sa-healthcare.iam.gserviceaccount.com\"\n",
```
Comment (codezizo) [inline imaging/ml/ml_codelab/breast_density_auto_ml.ipynb]: This is still needed. But instead of CHC-TCIA its $1, because we are doing an export still.

Comment (pavertomato) [reply imaging/ml/ml_codelab/breast_density_auto_ml.ipynb]: you mean https://github.com/GoogleCloudPlatform/healthcare/pull/443/files/eca5127e40dcf774337e224b993dcba1dec5ec0e#diff-f6a4270c35f32aba098eef15efc8f250R103 ?
service account have pubsub.admin(includes pubsub.publisher https://cloud.google.com/pubsub/docs/access-control) permissions and objectCreator

Comment (codezizo) [reply imaging/ml/ml_codelab/breast_density_auto_ml.ipynb]: service-${PROJECT_NUMBER}@gcp-sa-healthcare.iam.gserviceaccount.com(this is the one the CHC API uses when it exports vs ${PROJECT_NUMBER}-compute@developer.gserviceaccount.com,)

Comment (pavertomato) [reply imaging/ml/ml_codelab/breast_density_auto_ml.ipynb]: done

**Pull request merged**